src/FluentValidation.DependencyInjectionExtensions/ServiceCollectionExtensions.cs

```diff
@@ -23,17 +23,41 @@ namespace FluentValidation {
 	using System.Reflection;
 
 	public static class ServiceCollectionExtensions	{
+
 		/// <summary>
 		/// Adds all validators in specified assemblies
 		/// </summary>
 		/// <param name="services">The collection of services</param>
 		/// <param name="assemblies">The assemblies to scan</param>
-		/// <param name="lifetime">The lifetime of the validators. The default is transient</param>
-		/// <param name="filter">Optional filter that allows certain types to be skipped from registration.</param>
+		/// <param name="configExpression">Optional config object that contains factory registration information.</param>
 		/// <returns></returns>
-		public static IServiceCollection AddValidatorsFromAssemblies(this IServiceCollection services, IEnumerable<Assembly> assemblies, ServiceLifetime lifetime = ServiceLifetime.Transient, Func<AssemblyScanner.AssemblyScanResult, bool> filter = null) {

```

Comment:
Please re-add the overload which was removed as people actively use this. It an just be wrapper around one of the other overloads.